media/js/firefox/whatsnew/whatsnew-127-eu.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+/* eslint no-unused-vars: [2, { "varsIgnorePattern": "onYouTubeIframeAPIReady" }] */
```

Comment:
**Nit:** can this be removed, seems redundant?

Reply:
leftover copypasta 🍝